orca/templates/db/resources/usr/local/bin/mysql_init.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+
+set -e
+
+# Inspired by Percona's MySQL docker image, might share code snippets
+
+# shellcheck disable=SC1091
+. _liferay_common.sh
+
+function check_setup {
+	block_begin "Check if MySQL set up"
+
+	if [ -d /var/lib/mysql/data/mysql ];
+	then
+		block_finish "Check if MySQL is set up: DONE"
+
+		exit 0
+	else
+		block_finish "Check if MySQL is set up: NOT DONE"
+	fi
+}
+
+function datadir_init {
+	block_begin "Datadir initialization"
+
+	install -v -o mysql -g mysql -m 0700 -d "${HOME}/data" "${HOME}/log"
+
+	mysqld --initialize-insecure
+
+	block_finish "Datadir initialization"
+}
+
+function first_run_start {
```

Comment:
Recommending: start_first_run

Reply:
Now, I recall that Bryan said that functions should start with a verb.
However, in this case it is way more valuable to start with the object, because this way all the object related actions (functions) can be found at the same place so that it is safer and faster to work with it.

Anyways, function names have been changed.